### DISCUSSION TITLE
ClaimLog

### DISCUSSION THREAD

**Discussion by akdukaan** [GriefPrevention/GriefPrevention#2323] [Addons]
Keeps track of claim history including by creation/resize/deletion

https://modrinth.com/plugin/claimlog

**Comment by QarthO** [comment]
Is the source available for this?

**Reply by akdukaan** [reply]
https://github.com/akdukaan/ClaimLog

**Reply by TomLewis** [reply]
can you build and put the latest release on that github?

**Reply by akdukaan** [reply]
The latest release is 0.3.2 and you can download the built jar from the Spigot link in this conversation.

**Reply by TomLewis** [reply]
Nope, the discord CDN has removed it "This content is no longer available."

**Reply by akdukaan** [reply]
Oop thanks for letting me know, I've updated source on Github (turns out I had a 0.3.4 saved locally) and uploaded it to Modrinth at https://modrinth.com/plugin/claimlog
